CosumerProject/Services/AnomalyDetectionService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using ConsumerProject.models;
```

Comment:
the anomoly detection logic is decoupled to the signalrR notification, we can do it better, let's discuss that.